workflow/UX-and-design-proposals.md

```diff
@@ -0,0 +1,147 @@
+
+### Doc & Specs
+Spec docs with core engineers are also very important for the definition of the feature and to start a conversation around API, user stories and functionality.
+
+### Zeplin
+When ready for wireframes it can then move to Zeplin. Use tags and naming convention in a way that reflects the status of the project.
+
+### General direction
+We should work more on documents/specs to discuss things first, back and forth with engineers to discuss sketches about scope before moving to Zeplin for further feedback.
+
+
+## Zeplin/Miro status convention
+
+### Status & tags
+Status can be used to display the phase of the project so that people know you are looking for feedback reflecting that status, e.g.: ‘explorations’ (UX phase, asking for UX and scoping type of feedback, including implementation constraints and feasibility).
+
+### Status terminology:
+**Discovery**
+- Work on specs
+- Definition phase
+- Conceptual work
+- User interviews
+- Core engineers and stakeholders sync
+- Personas, user stories
+- Use cases, scenarios
+
+**Exploration**
+- First iterations of sketches/wireframes after spec & definition
+- Final UX proposal (last wireframe to conclude UX)
+
+**Visual**
+- Design stage
+- Feedback on the visual
+
+**Development**
+- This is ready to be implemented
+- Highlight implementation specs, annotation
+
+### Proposed tags, to be added for review stages
+We discussed reflecting the tag-naming convention from GitHub when asking for a certain type of feedback:
+- Review: UX needed
+- Review: design needed
+- Review: QA needed
+
+
+## Projects name
+
+Name your projects this way, using the year and the month of the cycle the project refers to:  “YY.MM Product - Project”, e.g. : “20.04 Snap - Progressive release”.
+
+## Sections name
+
+- Name your sections and your artboards so they make sense to others. And to you, in 6 months time.
+- Others need to know the step or phase, what they’re looking at, and what kind of thing it is:
+  “Iteration N. - Stage - thing”,  eg:
+  “1 - User flow - sharpies sketch from Warsaw workshop”
+  “4 - List and details - wires”
+  “Option B1 - Payment flow - visuals”
+- Add the status also to the title of the sections:
+  “[STATUS NAME] Iteration N - name-of-the-feature”
+	e.g. : “[EXPLORATION] Iteration III - Model detail views + add controller”
+- Always upload new screens into new sections, let’s not replace screens (except for quick fixes and typos).
+
+## Links to specs/docs/epics
+
+Do put links to specs, docs or issues/epics in the description field, so people can find their way back to the spec, and to the associated cards.
+
+
+### Who is it for?
+
+What feedback do you want? UX input, visual input, front-end input, engineering input etc.
+When uploading, think about the status of the project, what would you like feedback on may affect who you want to share it with it.
```

Comment:
When uploading, think about the status of the project, what would you like feedback on may affect who you want to share it with.